app/views/content/life-as-a-teacher/_directory-nav.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render Content::ImageBlockComponent.new(
+        title: "Life as a teacher",
+        title_color: "yellow",
+        image_path: "static/images/content/homepage/directory1.jpg"
+      ) do %>
+        <ul>
+          <li><a href="/life-as-a-teacher/pay-and-benefits">Pay and benefits</a></li>
+          <li><a href="/life-as-a-teacher/teaching-as-a-career">Teaching as a career</a></li>
+          <li><a href="/life-as-a-teacher/explore-subjects">Explore subjects you could teach</a></li>
+          <li><a href="/life-as-a-teacher/age-groups-and-specialisms">Age groups and specialisms</a></li>
+          <li><a href="/life-as-a-teacher/change-careers">Change careers to teaching</a></li>
+        </ul>
+      <% end %>
+      <%= render Content::ImageBlockComponent.new(
+        title: "Teacher stories",
+        title_color: "pink",
+        image_path: "static/images/content/homepage/directory2.jpg"
+      ) do %>
+        <ul>
+          <li><a href="/life-as-a-teacher/pay-and-benefits/bens-favourite-things-about-teaching">Ben's favourite thing about teaching</a></li>
```

Comment:
Should more stories be listed, now we have more?

Reply:
we've kept it to just a few for now as the titles are longer and will update them when we can see if it gets used